modules/cp4ba/variables.tf

```diff
@@ -65,26 +88,22 @@ variable "db2_user" {
   description = "User name defined in LDAP"
 }
 
-variable "db2_password" {
-  default     = "passw0rd"
-  description = "Password defined in LDAP"
+variable "db2_admin_user_password" {
+  description = "Db2 admin user password defined in LDAP"
 }
 
-variable "db2_host_name" {
-  default     = ""
-  description = "Host name of DB2 instance"
+variable "db2_host_address" {
+  description = "Host name for DB2 instance. Ignore if there is not an existing Db2."
 }
 
-variable "db2_host_port" {
-  default     = ""
-  description = "Port for DB2 instance"
+variable "db2_ports" {
```

Comment:
Is this an array of ports?  Or is it a single port number?  The name of the variable is inconsistent with the description.

Reply:
Changed to: 
variable "db2_ports" {
  description = "List of Port numbers of DB2 instances. Ignore if there is not an existing Db2."
}